docs/proposals/PROPOSAL_C.md

```diff
@@ -321,3 +267,38 @@ Gives response:
   "Chocolate Sprinkles Ice Cream"  
 }
 ```
+
+## Requirements
```

Comment:
I'd like to see a "yes|no|partial" for each entry to make a future rubric updates easier? See #34

Reply:
For the api section, I’d suggest we add client/server for how the filter is resolved. 
As we’re seeing new scenarios get unlocked, (https://github.com/opencontainers/image-spec/pull/903) efficient server side indexing and queries will be a linchpin for whether registries can actually succeed or fail.

Reply:
I was saying "partial" for anything that didn't have an efficient option in D, along with an explanation. Trying to find the right balance between easy to view/compare and descriptive.